app/src/main/java/com/example/umlandowallet/Start.kt

```diff
@@ -63,48 +65,42 @@ fun start(
         (System.currentTimeMillis() * 1000).toInt()
     )
 
-    // Read Channel Monitor state from disk
-    // Initialize the hashmap where we'll store the `ChannelMonitor`s read from disk.
-    // This hashmap will later be given to the `ChannelManager` on initialization.
-    var channelMonitors = arrayOf<ByteArray>()
-    if (serializedChannelMonitors != "") {
-        Log.i(LDKTAG, "LDK: initiating channel monitors...")
-        val channelMonitorHexes = serializedChannelMonitors.split(",").toTypedArray()
-        val channelMonitorList = ArrayList<ByteArray>()
-        channelMonitorHexes.iterator().forEach {
-            val channelMonitorBytes = it.toByteArray()
-            channelMonitorList.add(channelMonitorBytes)
-        }
-        channelMonitors = channelMonitorList.toTypedArray()
-    }
-
     // This is going to be the fee policy for __incoming__ channels. they are set upfront globally:
     val userConfig = UserConfig.with_default()
     val newChannelConfig = ChannelConfig.with_default()
-    newChannelConfig.set_forwarding_fee_proportional_millionths(10000)
-    newChannelConfig.set_forwarding_fee_base_msat(1000)
-    userConfig.set_channel_config(newChannelConfig)
+    newChannelConfig._forwarding_fee_proportional_millionths = 10000
```

Comment:
Yeah these are nice little refactorings that make the codebase more in line with Kotlin idioms. 👍 The transition that the compiler does between a Java setter and its Kotlin property equivalent makes the name of the property look a bit weird, but I think it's the better of the two options.